### DISCUSSION TITLE
Any plans to support PostGIS extension?

### DISCUSSION THREAD

**Discussion by gamesbook** [neondatabase/neon#1913] [Ideas]
PostGIS is the most important PG extension used by the geospatial community, and direct support of this would be a key consideration for using Neon - see also https://postgis.net/ and  https://www.timescale.com/blog/top-5-postgresql-extensions/

**Comment by hlinnaka** [comment]
Sure, we plan to support PostGIS. We did a quick experiment already to check that it works with neon, and it does.

We'd like to support any and all extensions, by allowing users to bring their own binaries. Not sure how we're going to do that yet; maybe allow users to provide their own docker image that includes the extensions they want, or something like that. But PostGIS is one of the most important ones, so we will definitely make that easy to install.

**Comment by acervantes23** [comment]
Hi @gamesbook,

You can now install PostGIS with the usual `CREATE EXTENSION postgis` statement! Please let us know if you are having any issues. Best of luck!
